### ISSUE TITLE
CDNJS pingdom uptime report data unavailable. No data since 05 June

### ISSUE THREAD

**Issue by altmind** [cdnjs/website#38]
http://stats.pingdom.com/htbkuegtgycv/1168714 displays question marks for last 7 days.

http://stats.pingdom.com/htbkuegtgycv/1168714/2014/06 displays that no data is vailable since 5 june.

Please enable uptime report.

**Comment by rclmenezes** [comment]
Looks like the uptime report is back down.

**Comment by altmind** [comment]
Yes, looks like http://stats.pingdom.com/htbkuegtgycv is no longer public

**Comment by thomasdavis** [comment]
Thanks guys!

hey @ryankirkman  can you take care of this one?

**Comment by ryankirkman** [comment]
Emailed pingdom. I'll keep you guys posted.
